Userland/Libraries/LibAudio/MP3Loader.cpp

```diff
@@ -5,6 +5,7 @@
  */
 
 #include "MP3Loader.h"
+#include "LibAudio/MP3Types.h"
```

Comment:
@kleinesfilmroellchen not sure if this include style keeps being used by clangd or what, but please either configure it to use `<>` style (if it can) or disable it. This include style keeps occurring in your PRs, so if it's an automated thing, let's avoid it.

Reply:
My new clangd config should prevent autoincludes. It is not yet configurable.